packages/jellyfish-api-core/src/category/masternode.ts

```diff
@@ -275,6 +313,7 @@ export interface MasternodeInfo {
   targetMultiplier?: number
   targetMultipliers?: number[]
   timelock?: number
+  rewardAddress: string
```

Comment:
The factual fact is this object is not the latest. https://github.com/JellyfishSDK/jellyfish/pull/1429 should be merged to main before this PR and it will update this object